apps/meteor/ee/client/sidebar/footer/SidebarFooterWatermark.tsx

```diff
@@ -1,16 +1,19 @@
-import { Box } from '@rocket.chat/fuselage';
+import { Box, Skeleton } from '@rocket.chat/fuselage';
 import { useTranslation } from '@rocket.chat/ui-contexts';
 import type { ReactElement } from 'react';
 import React from 'react';
 
-import { useIsEnterprise } from '../../../../client/hooks/useIsEnterprise';
+import { useLicense } from '../../../../client/hooks/useLicense';
 
 export const SidebarFooterWatermark = (): ReactElement | null => {
 	const t = useTranslation();
 
-	const { isLoading, isError, data } = useIsEnterprise();
+	const { data: { license, activeModules = [], trial: isTrial = false } = {}, isLoading, isError } = useLicense();
+	const [{ name: planName }] = license?.information?.tags ?? [{ name: 'Community' }];
```

Comment:
I suggest cleaning up this assignment: 
```suggestion
	const planName = license?.information?.tags && license.information?.tags[0]?.name || 'Community';
```